src/application/components/technologies/technology-sidebar/technology-sidebar.js

```diff
@@ -44,7 +45,6 @@ class TechnologySidebar extends Component {
   };
 
   addTechnology = event => {
-    event.preventDefault();
```

Comment:
Is this no longer needed? I think I saw this in the button.js file too.

Reply:
Yeah it's not needed anymore. I'm looking into why.